<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindDialog</class>
 <widget class="QDialog" name="FindDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>158</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>45</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>查找字符串：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FindPushButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>查找全部</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ReCheckBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>141</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>支持正则表达式</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="IgonreCheckBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>111</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>忽略大小写</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="FindLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="FindNextPushButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>40</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>查找下一个</string>
   </property>
  </widget>
  <widget class="QLabel" name="number">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
